### PR TITLE
CF - Checks - WorkSpaces

### PR DESCRIPTION
**Pull request by njgibbon**
Hello,

Adding checks for workspaces for CF as done for TF.

CF Docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-workspaces-workspace.html
TF Checks:
* https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/WorkspaceRootVolumeEncrypted.py
* https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/WorkspaceUserVolumeEncrypted.py

**License**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/cloudformation/checks/resource/aws/example_WorkspaceUserVolumeEncrypted/PASS.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  Pass:
+    Type: AWS::WorkSpaces::Workspace
+    Properties: 
+      UserName: test
+      BundleId: wsb-abc123456
+      DirectoryId: d-abc123456
+      UserVolumeEncryptionEnabled: True
```
Comment (nimrodkor) [inline tests/cloudformation/checks/resource/aws/example_WorkspaceUserVolumeEncrypted/PASS.yaml:9]: @njgibbon Shouldn't it be yaml boolean, i.e. `true`?

Comment (njgibbon) [reply tests/cloudformation/checks/resource/aws/example_WorkspaceUserVolumeEncrypted/PASS.yaml:9]: hey @nimrodkor  - Good point, good catch! I'll update when I have a bit of time spare.

Comment (njgibbon) [reply tests/cloudformation/checks/resource/aws/example_WorkspaceUserVolumeEncrypted/PASS.yaml]: @nimrodkor  Update now :-)

**Review by nimrodkor — COMMENTED** [review]
Thank you @njgibbon !

**Pull request merged**